src/OrchardCore.Modules/OrchardCore.BackgroundTasks/Controllers/BackgroundTaskController.cs

```diff
@@ -130,16 +152,19 @@ public async Task<IActionResult> Edit(string name)
             }
 
             var document = await _backgroundTaskManager.GetDocumentAsync();
-
-            var settings = GetSettings(task, document);
+            var defaultSettiongs = task.GetDefaultSettings();
```

Comment:
`defaultSettiongs` => `defaultSettings`
Idem in other places

Reply:
Fixed